Samples/Iperf/app_manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "SchemaVersion": 1,
+  "Name": "iperf3",
+  "ComponentId": "58981b9a-38be-4de2-9fba-7b00fe15a2c3",
```

Comment:
It looks like the component ID was cloned from the ext-iperf3 app manifest.  Is it worth changing, just to avoid risk of confusion for internal users of both?  I'm OK with not fixing this.

Reply:
Not sure duplicate component IDs matter in this case since we won't release ext-iperf externally.